home/daraz.py

```diff
@@ -14,7 +14,10 @@ def collect_items(browser):
             price = browser.find_element(By.XPATH, f'//div[starts-with(@data-qa-locator, "product-item")][{item_id}]/div[1]/div/div[2]/div[3]/span')
             image = browser.find_element(By.XPATH, f'//div[starts-with(@data-qa-locator, "product-item")][{item_id}]/div[1]/div/div[1]/div/a/img').get_attribute('src')
             link = title.get_attribute('href')
-
+            query_set = set(query.lower().split())
+            title_set = set(title.text.lower().split())
```

Comment:
Use regex instead of set
```py
pattern = query.replace(" ", ".+")
pattern = f".+{pattern}.+"
if not re.match(pattern, title.text, re.IGNORECASE):
    continue
```